plugins/woocommerce/i18n/currency-info.php

```diff
@@ -3,950 +3,1080 @@
  * Currency formatting information
  *
  * @package WooCommerce\i18n
- * @version 5.7.0
+ * @version 8.1.0
```

Comment:
Looking at the release calendar, his should probably be:

```suggestion
 * @version 8.2.0
```

Reply:
Thank you! I'll update both files.

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/40101/commits/3de8ed8c55df591024db9f25148e908d87376228.